src/didactUriCompletionItemProviderAsciiDoc.ts

```diff
@@ -0,0 +1,96 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+import * as vscode from 'vscode';
+import {didactProtocol, DidactUriCompletionItemProvider} from './didactUriCompletionItemProvider';
+import * as extensionFunctions from './extensionFunctions';
+
+export class DidactUriCompletionItemProviderAsciiDoc extends DidactUriCompletionItemProvider {
+
+	protected getCommandCompletionItems() : vscode.CompletionItem[] {
+		let completions : vscode.CompletionItem[] = [];
+
+		// Terminal commands
+		super.startTerminalWithNameCompletion("Start Terminal with Name", completions);
```

Comment:
comment for another PR as it was already done like that:
what about making the method returning a list of completions instead of passing the completions list as a parameter and modify it inside?
it will avoid to use the "Output Arguments" pattern which is advised to be avoided by Robert C. Martin in "Clean Code" book (Chapter 3)

Reply:
We can definitely look at making this better in another PR